index.html

```diff
@@ -175,16 +175,16 @@ <h5 class="h5">CONTACT US</h5>
            </p>
         </div>
         <!-- start form -->
-        <form action="https://formspree.io/f/mnqlvaya" class="footerform" method="POST">
+        <form action="https://formspree.io/f/xbjqperk" class="footerform" method="POST">
             <div class="formdiv">
                 <label for="name"></label>
-                <input type="text" name="name" id="name" class="formname" placeholder="Text input" maxlength="20" required>
+                <input type="text" name="name" id="name" class="formname" placeholder="text" maxlength="30" required>
                 <label for="email"></label>
-                <input type="email" name="email" id="email" class="formemail" placeholder="brahimdidi@gmail.com" maxlength="30" required>
+                <input type="email" name="email" id="email" class="formemail" placeholder="text" maxlength="30" required>
             
-            <textarea name="message" class="formmessage" id="message"  maxlength="400" placeholder="Enter text here" required></textarea>
-            <button class="seeproject2" type="submit">See Project</button>
+            <textarea name="message" class="formmessage" id="message"  maxlength="500" placeholder="text" required></textarea>
```

Comment:
- [ ] Please consider using an appropriate placeholder for the input fields and textarea, I suggest `name` , `name@email.com`, and `enter your message` respectively 🙂

Reply:
it is required to be text